aspnetcore/security/authentication/certauth.md

```diff
@@ -206,3 +206,243 @@ Complete the following steps In IIS Manager:
 ### Azure and custom web proxies
 
 See the [host and deploy documentation](xref:host-and-deploy/proxy-load-balancer#certificate-forwarding) for how to configure the certificate forwarding middleware.
+
+### Use certificate authentication with the X-ARR-ClientCert header
```

Comment:
I think this would be better titled as Using certificate authentication in Azure Web Applications. The header name is too technical for a title.

Reply:
fixed